code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -88,7 +125,12 @@
 			world << pick(creepyasssounds)
 		else
 			lastsinPerson = world.time
-			var/mob/living/carbon/human/sinPerson = pick(player_list)
+			var/mob/living/carbon/human/sinPerson = pick(living_mob_list)
+			var/depth = living_mob_list.len + 1 // just in case

```

Comment:
Combining both lists without duplicates would be |
If you don't care about duplicates, you use +